src/bento/navigationBarData.js

```diff
@@ -77,8 +81,9 @@ export const navBarData = [
 ];
 
 export const navBarCartData = {
-  cartLabel: 'MY Files',
+  cartLabel: '',
```

Comment:
Is it intended to be empty

Reply:
Yes as per new design this is moving to right side of icon.